script.js

```diff
@@ -47,3 +47,226 @@ navLink3.addEventListener('click', () => {
+const aboutPage = document.querySelector('#about-me');
+const footer = document.querySelector('footer');
+const popupBtn = document.querySelectorAll('.resume');
+const section = document.createElement('section');
+const body = document.querySelector('body');
+
+function closeMenu() {
+  mobileBg.style.display = 'none';
+  landingPage.style.filter = 'blur(0)';
+  aboutPage.style.filter = 'blur(0)';
+  footer.style.filter = 'blur(0)';
+  projectsPage.style.filter = 'blur(0)';
+  section.style.display = 'none';
+}
+
+function closeMenuDt() {
+  desktopBg.style.display = 'none';
+  landingPage.style.filter = 'blur(0)';
+  aboutPage.style.filter = 'blur(0)';
+  footer.style.filter = 'blur(0)';
+  projectsPage.style.filter = 'blur(0)';
+  section.style.display = 'none';
+}
+
+if (mediaQuery.matches) {
+  popupBtn.forEach((btn) => {
+    btn.addEventListener('click', () => {
+      window.scroll({ top: 0, left: 0 });
+      mobileBg.style.display = 'block';
+      section.innerHTML = pj1;
+      section.setAttribute('id', 'pj1');
+      landingPage.style.filter = 'blur(9px)';
+      aboutPage.style.filter = 'blur(9px)';
+      footer.style.filter = 'blur(9px)';
+      projectsPage.style.filter = 'blur(9px)';
+      body.append(section);
+      section.style.display = 'flex';
+      const closeBtn = document.querySelector('.close-btn');
+      closeBtn.addEventListener('click', closeMenu);
+    });
+  });
+} else {
+  popupBtn.forEach((btn) => {
+    btn.addEventListener('click', () => {
+      window.scroll({ top: 0, left: 0 });
+      desktopBg.style.display = 'block';
+      section.innerHTML = pj1Dt;
+      section.setAttribute('id', 'pj1-dt');
+      landingPage.style.filter = 'blur(9px)';
+      aboutPage.style.filter = 'blur(9px)';
+      footer.style.filter = 'blur(9px)';
+      projectsPage.style.filter = 'blur(9px)';
+      body.append(section);
+      section.style.display = 'block';
+      const closeBtnDt = document.querySelector('#close-btn-dt');
+      closeBtnDt.addEventListener('click', closeMenuDt);
+    });
+  });
+}
+
+/*validation Section*/
+// const email = document.getElementById('email');
+// const error = document.querySelector('.error');
+// const isLowerCase = (str) => str === str.toLowerCase();
+// document.getElementById('submit').addEventListener('submit', (event) => {
+//   if (!isLowerCase(email)) {
+//     event.preventDefault();
+//     error.innerHTML = 'Please enter a lowercase email';
+//   }
+// });
+
+document.querySelector('#submit').addEventListener('click', (event) => {
+  const email = document.getElementById('email').value;
+  const error = document.querySelector('.error');
+  const isLowerCase = (str) => str === str.toLowerCase();
+  if (!isLowerCase(email)) {
+    event.preventDefault();
+    error.innerHTML = 'Please write a lowercase email';
+    error.style.color = 'red';
+    setTimeout(() => {
+      error.innerHTML = '';
+    }, 5000);
+  }
+});
+
+
+
```

Comment:
- [ ] I think it would be better if you remove the unnecessary empty lines in your code to make it cleaner